examples/langchain-typescript-selfqueryingretrieval/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Can we cut this down to a minimal `tsconfig.json`?

Reply:
yup